git_cli_vs_gui/git_cli_vs_gui.md

```diff
@@ -0,0 +1,375 @@
+<!--
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+current_version_description: Initial version
+module_type: standard
+docs_version: 3.0.0
+language: en
+narrator: US English Female
+mode: Textbook
+title: Git Command Line Interface versus Graphical User Interface
+comment:  Compare the two ways of interacting with Git to decide which is best for you.
+long_description: You can use Git using a command line interface (CLI) and typing commands, and you can also use Git using graphical user interface software which is optimized for point-and-click.  This module will compare and contrast these two options to help you decide how to use Git in your work.
+estimated_time_in_minutes: 45
+
+@pre_reqs
+Learners should know what version control is and why it's important.  To acquire basic knowledge of version control, we recommend our [Introduction to Version Control](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/git_intro/git_intro.md#1) module.  No previous experience with git is required.
+@end
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Explain what a **graphical user interface** is and the advantages and limitations of a GUI
+- Explain what a **command line interface** is and the advantages and limitations of a CLI
+- Name one major Git graphical user interface 
+
+@end
+
+good_first_module: false
+coding_required: false
+
```

Comment:
Agree re: your placement in the sequence, but I'm not sure about the point of your question -- is there another "sequence" metadata thing I'm missing?  Or do you mean maybe I should consider "check out our next module"... in "intro" and otherwise insinuate sequence?  @franzenr @drelliche  since both of you are interested in metadata and sequencing

Reply:
I think the question is whether this should formally be part of the Git Basics sequence, in which case it would be labeled with that sequence name in the metadata and have a "previous sequential module" bit of metadata as well, and we would also have to change same bit of metadata for the next modules in the sequence, which I believe are setting up Git in Mac/Linux and Windows.

Reply:
Oh WAIT I'm a silly goose never mind I see the sequence metadata now that I'm looking in docs.md.  Disregard!